signupsignin.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>good.teens</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="signupsignin.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+        integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+</head>
+
+<body>
+    <script src="signupsignin.js"></script>
+    <script src="/__/firebase/7.15.1/firebase-app.js"></script>
```

Comment:
hm...where those paths are pointing to? If you would open a developer tool in your browser you would see a lot of FILE_NOT_FOUND errors because browser trying to find them in your project. But you should take them from a server like that:
```js
    <script src="https://www.gstatic.com/firebasejs/7.15.1/firebase-app.js"></script>
    <script src="https://www.gstatic.com/firebasejs/7.15.1/firebase-analytics.js"></script>
    <script src="https://www.gstatic.com/firebasejs/7.15.0/firebase-auth.js"></script>
    <script src="https://www.gstatic.com/firebasejs/7.15.0/firebase-firestore.js"></script>
```

Reply:
More on top: 
I think you forgot what I mentioned that you need to use 'setup with CDN' - [setup](https://firebase.google.com/docs/web/setup#from-the-cdn)

so what you need to do is to use this:
```html
    <script src="https://www.gstatic.com/firebasejs/7.15.1/firebase-app.js"></script>
    <script src="https://www.gstatic.com/firebasejs/7.15.1/firebase-analytics.js"></script>
    <script src="https://www.gstatic.com/firebasejs/7.15.0/firebase-auth.js"></script>
    <script src="https://www.gstatic.com/firebasejs/7.15.0/firebase-firestore.js"></script>
    <script>
        const firebaseConfig = {
            apiKey: "AIzaSyBbuvg-wCgz30zyD4zETU0HiHiC5_FulWc",
            authDomain: "good-teens.firebaseapp.com",
            databaseURL: "https://good-teens.firebaseio.com",
            projectId: "good-teens",
            storageBucket: "good-teens.appspot.com",
            messagingSenderId: "260739514487",
            appId: "1:260739514487:web:8197a36e539f090f2ec672",
            measurementId: "G-0WBRPFKT5Z"
        };

        // Initialize Firebase
        firebase.initializeApp(firebaseConfig);
    </script>
```

and after that you would have your application ready to work with firebase.

Reply:
and one more thing: 
in firebaseConfig - your `appId` looks strange you can double check it in your fireBase webpage.

Reply:
Yes, I did 'setup with CDN' according to the page. I will do it now again to see if anything changes. 
appID: 1:260739514487:web:8197a36e539f090f2ec672 - i know, looks weird.
![obrázok](https://user-images.githubusercontent.com/62797218/84770292-90c6d280-afd7-11ea-8c5a-4f78283fdf9f.png)